BT-Tracking/Services/Background/BackgroundService.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  BackgroundService.swift
+//  eRouska Dev
+//
+//  Created by Lukáš Foldýna on 17/08/2020.
+//  Copyright © 2020 Covid19CZ. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RealmSwift
+import BackgroundTasks
+
+struct BackgroundService {
+
+    let exposureService: ExposureServicing
+    let reporter: ReportServicing
+
+    let taskIdentifier = BackgroundTaskIdentifier.exposureNotification
```

Comment:
Na místech kde používáš tuto proměnou bych používal přímo `BackgroundTaskIdentifier.exposureNotification`. Plete se to s parametrem metody.